projects/aca-playwright-shared/src/page-objects/components/adf-info-drawer.component.ts

```diff
@@ -36,4 +36,36 @@ export class AdfInfoDrawerComponent extends BaseComponent {
   public getIdField = (labelText: string) => this.getChild('[data-automation-id="library-id-properties-wrapper"]', { hasText: labelText });
   public getVisibilityField = (labelText: string) => this.getChild('[data-automation-id="library-visibility-properties-wrapper"]', { hasText: labelText });
   public getDescriptionField = this.getChild('[data-automation-id="library-description-properties-wrapper"] textarea');
+  public propertiesTab = this.getChild('.adf-info-drawer-tab').nth(0);
+  public commentsTab = this.getChild('.adf-info-drawer-tab').nth(1);
+  public commentInputField = this.getChild('mat-form-field');
+  public commentsHeader = this.getChild('#comment-header');
+  public addCommentButton = this.getChild('[data-automation-id="comments-input-add"]');
+  public commentsList = this.getChild('.adf-comment-list-item');
+  public commentUsername = this.getChild('.adf-comment-user-name');
+  public commentTextContent = this.getChild('.adf-comment-message');
+  public commentTimestamp = this.getChild('.adf-comment-message-time');
+  public commentProfileIcon = this.getChild('.adf-comment-user-icon');
+
+
+  async checkCommentsHeaderCount() {
+    const commentsCountTextContent: string = await this.commentsHeader.textContent();
```

Comment:
typescript deduces type automatically if it has some context, in this case it can be deduced based on returned type of textContent function, so no need to set type manually for variables: 
<img width="695" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/384409e0-f6ef-48c3-a837-06e7b948ac44">

Reply:
Will be fixed in the next push